### DISCUSSION TITLE
Narrow no-break space

### DISCUSSION THREAD

**Discussion by meikezo** [guilryder/clavier-plus#35] [Q&A]
Hi,
thanks a lot for sharing this easy-to-use application!
I would like to add a hotkey to print a narrow no-break-space. Now I cannot really see what the characters 128-160 do and whether one of them is a narrow no-break-space.
If not, maybe there is a possibility to enter unicode-characters?
Thanks in advance for your help!

**Comment by guilryder** [comment]
Clavier+ can write Unicode text, assuming the receiving software supports Unicode too. Refer to the documentation for how to create text shortcuts. Internet search engines can hopefully answer your question about characters 128-160, as I didn't understand it.
